plugins/InspectionGadgets/InspectionGadgetsAnalysis/src/com/siyeh/ig/controlflow/UnnecessaryDefaultInspection.java

```diff
@@ -165,7 +164,7 @@ else if (defaultStatement instanceof PsiDefaultCaseLabelElement) {
         if (!isOnTheFly()) {
           return;
         }
-        highlightType = INFORMATION;
+        highlightType = LIKE_UNUSED_SYMBOL;
```

Comment:
INFORMATION, means here that there would be no highlighting and only fix would be provided. To change the highlighting you need to update line 170. Here, nothing can be changed